ports/openldap/vcpkg.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "openldap",
+  "version": "2.5.13",
+  "description": "OpenLDAP Software is an open source implementation of the Lightweight Directory Access Protocol.",
+  "homepage": "https://www.openldap.org/software/",
+  "license": "OLDAP-2.8",
+  "supports": "!windows, (mingw & !x86)",
+  "dependencies": [
+    "openssl",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
```

```suggestion
    "openssl"
```
The `vcpkg_cmake_xx` function is not used in portfile.cmake.

---

Removed

---

You need to remove it again. I missed that.